.github/workflows/android-cd.yml

```
@@ -56,6 +58,30 @@ jobs:
                     echo "Version extracted from toml: v${VERSION}"
                 id: extract_version
 
+            -   name: Generate Firebase Release Note
+                id: firebase_release_note
+                run: |
```

actionlint (1.7.4)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)